eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/service/UserService.java

```diff
@@ -50,7 +50,7 @@ public UserResponseDTO createUser(UserRequestDTO userRequestDTO) {
   public UserResponseDTO updateUser(Long id, UserRequestDTO userRequestDTO) {
     User user = userRepository.findById(id)
         .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
-    if (userRequestDTO.getPassword() != null
+    if (userRequestDTO.getPassword() != null && !userRequestDTO.getPassword().equals("")
```

Comment:
Code smell: string összehasonlításban először a literált írjuk. Ebben az esetben pont nincs belőle probléma, mert rácsekkolsz előtte, hogy null-e a password, de ha megfordítod az equals-t, akkor az el is hagyható.
https://stackoverflow.com/questions/24656018/string-literal-expressions-should-be-on-the-left-side-of-an-equals-comparison